tests/test_tokenization_common.py

```diff
@@ -83,6 +83,7 @@ class TokenizerTesterMixin:
     from_pretrained_kwargs = None
     from_pretrained_filter = None
     from_pretrained_vocab_key = "vocab_file"
+    test_seq2seq = True
```

Comment:
Introducing this new argument for tokenizers where we don't want to test `prepare_seq2seq_batch` since they now all gain that method (temporarily).